packages/sdk/src/modules/storage/index.ts

```diff
@@ -0,0 +1,248 @@
+            unsubscribe,
+        };
+        
+    } catch (error) {
+        throw new StorageError(`${error}`);
+        }
+    }
+
+
+    /**
+     * Removes the itemType in peaqStorage
+     */
+
+    public async removeItem(
+        options: RemoveItemOptions,
+        statusCallback?: (result: ISubmittableResult) => void | Promise<void>
+     ): Promise<RemoveItemResult> {
+        try {
+            const api = this._getApi();
+
+            const { itemType, seed = ''} = options;
+            if (!itemType) throw new ItemTypeError('Item Type name is required');
+            if (seed !== '') this._checkSeed(seed);
+
+            const keyPair = this._metadata?.pair || this._getKeyPair(seed);
+            const attributeExtrinsic = api.tx?.['peaqStorage']?.['removeItem'](
+                itemType
+              );
+              const nonce = await this._getNonce(keyPair.address);
+              const eventData = await this._newSignTx({nonce, address: keyPair, extrinsics: attributeExtrinsic});
+              // await attributeExtrinsic.signAsync(keyPair, { nonce });
+              const unsubscribe = await attributeExtrinsic.send((result) => {
+                statusCallback &&
+                  statusCallback(result as unknown as ISubmittableResult);
+              });
+          
+              // successfully removed the did of name __ from address machine address __
+              return {
+                log: `Successfully removed the storage item type ${itemType} from address ${keyPair.address}`,
+                block_hash: eventData[0]?.blockHash as unknown as CodecHash,
+                unsubscribe,
+              };
+
+        } catch (error) {
+            throw new StorageError(`${error}`);
+        }
+     }
+
+     public async getItem(options: GetItemOptions): Promise<GetItemResult | null> {
+       try {
+        const api = this._getApi();
+
+        const { itemType, address = '' } = options;
+        if (!itemType) throw new ItemTypeError('Item Type name is required');
+        if (address !== '') this._checkAddress(address);
+
+        const accountAddress = address || this._metadata?.pair?.address;
+        if (!accountAddress) throw new StorageAddressError('Address is required');
+
+            const { hashed_key } = createStorageKeys([
+                {
+                  value: accountAddress,
+                  type: CreateStorageKeysEnum.ADDRESS,
+                },
+                { 
+                    value: itemType, type: CreateStorageKeysEnum.STANDARD
+                },
+            ]);
+            const item = (await api.query?.['peaqStorage']?.['itemStore'](
+                hashed_key
+            ));
+            if (!item || item.isStorageFallback) return null;
+            // is toHuman acceptable here? What if a simple string is not passed?
+            return {
+                log: `${item.toHuman()}`,
+            };
```

Comment:
Please return the item just as it was stored on chain.
Don't parse it using `toHuman`.